BUILD.md

```diff
@@ -71,6 +71,15 @@ contains the Vulkan API definition files (registry) that are required to build
 the loader. You must also take note of the headers install directory and pass
 it on the CMake command line for building this repository, as described below.
 
+#### Windows Driver Kit (WDK)
+
+On Windows builds, the loader needs to have a WDK installed. Microsoft provides
+[WDK releases](https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk),
+including several old releases. The installed WDK must be at least version 1709.
```

Comment:
Is that right? I thought a newer version was required. 1803 seems to be the oldest that contains either of these symbols: `KMTQAITYPE_QUERYREGISTRY`, `D3DDDI_QUERYREGISTRY_ADAPTERKEY`. I do not see these symbols in 1709.

Reply:
Here's evidence suggesting the first version is 1803:

```$ find . -iname d3dkmthk\.h
./10240/Include/10.0.10240.0/km/d3dkmthk.h
./10586/Include/10.0.10586.0/km/d3dkmthk.h
./14393-5/Include/10.0.14393-5.0/km/d3dkmthk.h
./15063/Include/10.0.15063.0/km/d3dkmthk.h
./16299-15/Include/10.0.16299-15.0/km/d3dkmthk.h
./17134/Include/10.0.17134.0/km/d3dkmthk.h
./17763/Include/10.0.17763.0/km/d3dkmthk.h
./18362/Include/10.0.18362.0/km/d3dkmthk.h

$ find . -iname d3dkmthk\.h -print0 | xargs -0 grep KMTQAITYPE_QUERYREGISTRY
./17134/Include/10.0.17134.0/km/d3dkmthk.h:     KMTQAITYPE_QUERYREGISTRY           = 48,
./17763/Include/10.0.17763.0/km/d3dkmthk.h:     KMTQAITYPE_QUERYREGISTRY           = 48,
./18362/Include/10.0.18362.0/km/d3dkmthk.h:     KMTQAITYPE_QUERYREGISTRY           = 48,
```

where
```
 |   subdir |   build# | codename | version                     |
 |----------+----------+----------+-----------------------------|
 |    10240 |    10240 | TH1      | 1507                        |
 |    10586 |    10586 | TH2      | 1511 (November Update)      |
 |  14393-5 |    14393 | RS1      | 1607 (Anniversary Update)   |
 |    15063 |    15063 | RS2      | 1703 (Creators Update)      |
 | 16299-15 | 16299.15 | RS3      | 1709 (Fall Creators Update) |
 |    17134 |    17134 | RS4      | 1803 (April 2018 Update)    |
 |    17763 |    17763 | RS5      | 1809 (October 2018 Update)  |
 |    18362 |    18362 | 19H1     | 1903 (April 2019 Update)    |
```

Reply:
In case this helps, here's a summary of OS versions: https://en.wikipedia.org/wiki/Windows_10_version_history

And info about availability date, and end of service date:
https://docs.microsoft.com/en-us/windows/release-information/